tests/Tests/600__ai_apps/660__openvino/660__openvino.robot

```diff
@@ -12,37 +12,55 @@ Suite Teardown  OpenVino Suite Teardown
 *** Variables ***
 ${openvino_appname}           ovms
 ${openvino_container_name}    OpenVINO
-${openvino_operator_name}    OpenVINO Toolkit Operator
+${openvino_operator_name}     OpenVINO Toolkit Operator
 
 *** Test Cases ***
 Verify OpenVino Is Available In RHODS Dashboard Explore Page
-  [Tags]  ODS-258  Smoke  Sanity
-  Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
-  Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
-  Wait for RHODS Dashboard to Load
-  Verify Service Is Available In The Explore Page    OpenVINO
-  Verify Service Provides "Get Started" Button In The Explore Page    OpenVINO
+   [Tags]  ODS-258  Smoke  Sanity
+   Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+   Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+   Wait for RHODS Dashboard to Load
+   Verify Service Is Available In The Explore Page    OpenVINO
+   Verify Service Provides "Get Started" Button In The Explore Page    OpenVINO
 
 Verify Openvino Operator Can Be Installed Using OpenShift Console
    [Tags]   Tier2
    ...      ODS-675
+   ...      ODS-652
    ...      ODS-495
    ...      ODS-1236
-   [Documentation]  This Test Case Installed Openvino operator in Openshift cluster
+   [Documentation]  This Test Case Installed Openvino operator in Openshift cluster,
+   ...              Cheks whether RHODS notification shows the Notebook images are running after installing openvino
    ...               and Check and Launch AIKIT notebook image from RHODS dashboard
    Check And Install Operator in Openshift    ${openvino_operator_name}   ${openvino_appname}
    Create Tabname Instance For Installed Operator        ${openvino_operator_name}       Notebook    redhat-ods-applications
    Wait Until Keyword Succeeds    900  1     Check Image Build Status   Complete     openvino-notebook
+   ${build_list} =    Get The Labels Of Build    namespace=redhat-ods-applications    build_search_term=openvino-notebook
+   Run Keyword And Return Status    Should Contain    ${build_list}[0]    opendatahub.io/build_type\n=\nnotebook_image
    Go To RHODS Dashboard
+   RHODS Notification Drawer Should Contain    message=Notebook images are building
    Verify Service Is Enabled          ${openvino_container_name}
    Verify JupyterHub Can Spawn Openvino Notebook
    [Teardown]   Uninstall Openvino Operator
 
-
 *** Keywords ***
+Get The Labels Of Build
```

Comment:
this could be useful for other TCs. Would you move it to a common resource file?
Then, I would also see if it is faster to use openshiftlibrary rather than UI driver for getting the info here. Could you do some attempts and compare the time?

Reply:
Done, moved to the file
Will try to compare